Common/Constants.cs

```diff
@@ -90,5 +90,7 @@ public static TimeSpan MaxAllowedExecutionTime
 
         //Setting for VC++ for node builds
         public const string VCVersion = "2015";
+
+        public const string CrashDumps = "CrashDumps";
```

Comment:
Maybe this folder should be just "dumps"? If we decide to write other types of dumps there in the future?